neo/Ledger/MemoryPool.cs

```diff
@@ -361,7 +361,8 @@ internal void UpdatePoolForBlockPersisted(Block block, Snapshot snapshot)
                 if (policyChanged)
                 {
                     foreach (PoolItem item in _unverifiedSortedTransactions.Reverse())
-                        _system.Blockchain.Tell(item.Tx, ActorRefs.NoSender);
+                        if(item.Tx.FeePerByte <= NativeContract.Policy.GetFeePerByte(snapshot))
```

Comment:
`NativeContract.Policy.GetFeePerByte(snapshot)` must be cached outside the loop

Reply:
Sharrrgons's brother, that is true, nice catch.